requirements.txt

```diff
@@ -6,11 +6,13 @@ cloudpickle>=2.0.0
 dask[distributed]>=2022.6.0
 fastapi>=0.93.0
 furl>=2.1.3
+natsort>=8.4.0
```

Comment:
@wjcunningham7 is this being used anywhere ?

Reply:
It's used to sort the names of the dask workers when running `covalent cluster --status`